taichi/ir/stmt_op_types.h

```diff
@@ -84,7 +84,13 @@ enum class SNodeOpType : int {
 
 std::string snode_op_type_name(SNodeOpType type);
 
-enum class TextureOpType : int { sample_lod, fetch_texel, undefined };
+enum class TextureOpType : int {
+  sample_lod,
+  fetch_texel,
+  load,
+  store,
+  undefined
```

Comment:
I think the convention is to put `undefined` in the first place? (Also we should rename this to `kXx` style later)